### PR TITLE
Update parameters for flexvolume-driver

### PR DESCRIPTION
**Pull request by shivamerla**

### CODE REVIEW

File: helm/charts/hpe-flexvolume-driver/README.md [hunk 1]
```diff
@@ -26,8 +26,11 @@ The following table lists the configurable parameters of the HPE FlexVolume Driv
 | fsType             | Type of file to format volumes with (ext4, ext3, xfs, btrfs).                                        | xfs         |
 | mountConflictDelay | Wait this long (in seconds) before forcefully taking over a volume from an isolated or crashed node. | 150         |
 | flavor             | Kubernetes distribution specific tweaks. Currently only needed for `openshift`.                      | kubernetes           |
-| kubeletPath        | This is the directory where the kubelet lives. May differ between Kubernetes distributions.          | /var/lib/kubelet     |
+| podsMountDir       | This is the directory where the kubelet bind mounts the volume for pod. May differ between Kubernetes distributions.          | /var/lib/kubelet/pods     |
```
Comment (rgcostea) [inline helm/charts/hpe-flexvolume-driver/README.md]: for pods.

File: helm/charts/hpe-flexvolume-driver/README.md [hunk 2]
```diff
@@ -26,8 +26,11 @@ The following table lists the configurable parameters of the HPE FlexVolume Driv
...
 | flexVolumeExec     | This is the path where the FlexVolume binary gets installed on the host.                             | default     |
+| storageClass.name  | The name to assign the created StorageClass .                                          | hpe-standard |
```
Comment (rgcostea) [inline helm/charts/hpe-flexvolume-driver/README.md]: white space between Storage Class and "."

File: helm/charts/hpe-flexvolume-driver/README.md [hunk 3]
```diff
@@ -43,6 +46,7 @@ Applicable to Red Hat OpenShift 3.10 and 3.11. 4.x is not supported.
 | Key        | Value                     | Description                                                                        |
 |------------|---------------------------|------------------------------------------------------------------------------------|
 | flavor     | openshift                 | nodeSelector tweaks to prevent provisioner to run on an infra node.                |
+| podsMountDir | /var/lib/origin/openshift.local.volumes       | This is the directory where the kubelet bind mounts the volume for pod.            |
```
Comment (rgcostea) [inline helm/charts/hpe-flexvolume-driver/README.md]: for pods.

File: helm/charts/hpe-flexvolume-driver/README.md (hunk 1 above)
Comment (shivamerla) [reply helm/charts/hpe-flexvolume-driver/README.md]: sure, will fix these with next update..

**Pull request merged**